Apps/Src/main.c

```diff
@@ -14,6 +14,10 @@ int main() {
 	
 	OS_ERR err;
 
+	OSInit(&err);
+
+	// ASSERT err
```

Comment:
Are we missing the assert function?

Reply:
The assert wasn't merged yet when I did this but now I see the rtos branch now has it. I'll update it